setup.py

```diff
@@ -104,6 +104,7 @@
         'pysynphot',
         'lxml',
         'PyPDF2',
+        'PyTables'
```

Comment:
THere is a comma missing after 'PyTables' which is causing a build error.

Reply:
Resolved in commit [24e4ba1](https://github.com/spacetelescope/drizzlepac/pull/682/commits/24e4ba1a02b63470f1a9e08a050dc8a42274eb02)